pytest_tests/testsuites/services/s3_gate/test_s3_gate.py

```diff
@@ -133,7 +134,7 @@ def test_s3_buckets(self, client_shell):
                 s3_gate_bucket.head_bucket(self.s3_client, bucket_1)
 
     @allure.title("Test S3 Object API")
-    @pytest.mark.sanity
+    @pytest.mark.nightly
```

Comment:
do we need this mark, if all class was marked as nightly ?

Reply:
ok, I kept old mark name: "sanity"